generate/write-to-json-file.js

```diff
@@ -0,0 +1,45 @@
+import path from 'path'
+import fs from 'fs'
+
+const JsonStringifyParameters = {
+  replacer: null,
+  space: 4,
+}
+
+export function writeJsonArray(arrayNewItems, config, applyFilter = false) {
+  const DEST_JSON_FILE = path.join(__dirname, config.destDirectoryFromHere, `${config.destFileName}.json`)
+  const allFileContent = arrayNewItems.map(item => {
+    return applyFilter ? filterProperties(config.propertiesToFilter, item) : item
+  })
+  writeToFile(DEST_JSON_FILE, stringify(allFileContent))
+}
+
+export function writeJsonObject(newItem, config, applyFilter = false) {
+  const DEST_JSON_FILE = path.join(__dirname, config.destDirectoryFromHere, `${config.destFileName}.json`)
+  const itemToCopy = applyFilter ? filterProperties(config.propertiesToFilter, newItem) : newItem
+  writeToFile(DEST_JSON_FILE, stringify(itemToCopy))
```

Comment:
By default, JSON is written with no space and on a single line. 
The reason behind : in fact it is just because it breaks my Atom when I read an episode json file (*single episode json: there is no parameter filter, so, with transcripts it can fast become a super huge line*).

**Updated to simple:** 
```javascript 
writeToFile(DEST_JSON_FILE, JSON.stringify(itemToCopy))
```